src/qt/locale/bitcoin_zh-Hans.ts

```diff
@@ -67,12 +67,12 @@
     </message>
     <message>
         <source>These are your Bitcoin addresses for sending payments. Always check the amount and the receiving address before sending coins.</source>
```

Comment:
I'm not an expert in this, but I think the source phase with "Bitcoin" should also be changed into "Dogecoin" in order to make the translation work. Please correct me it I'm wrong